engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -122,8 +121,8 @@
    * This represents an element that can be referenced in a DT property.
    */
   protected enum StramElement {
-    APPLICATION("application"), GATEWAY("gateway"), TEMPLATE("template"), OPERATOR("operator"),STREAM("stream"), PORT("port"), INPUT_PORT("inputport"),OUTPUT_PORT("outputport"),
-    ATTR("attr"), PROP("prop"),CLASS("class"),PATH("path"),UNIFIER("unifier");
+    APPLICATION("application"), GATEWAY("gateway"), TEMPLATE("template"), OPERATOR("operator"), STREAM("stream"), PORT("port"), INPUT_PORT("inputport"), OUTPUT_PORT("outputport"),
+    ATTR("attr"), PROP("prop"), CLASS("class"), PATH("path"), UNIFIER("unifier"), MODULE("module");
```

Comment:
I thought we wanted to use the same configuration file format? As far as the user is concerned, a module is just another operator.